### ISSUE TITLE
Absolute depth

### ISSUE THREAD

**Issue by liu-vis** [KexianHust/Structure-Guided-Ranking-Loss#2]
Dear author,

Thank you for your great work!
Could you please tell me how to obtain the absolute depth from the output of your model?
I printed out the "max_d" and "min_d" that are computed by "min_d, max_d = depth.min(), depth.max()",
I found that min_d could be a negative value, specifically, for example:
min_d =  -2.64,  max_d = 5.57 for "examples/0SpJOOTH7R4_144577767_image.jpg".
I thus suppose that the output of the CNN model is not the absolute depth.
Thank you for your time, and looking forward to your reply.

**Comment by KexianHust** [comment]
Yes, you're right!  This model is used to estimate the relative depth from a single image.

**Comment by liu-vis** [comment]
Dear author,
Thank you for answering the question!
Could you please let me know how did you transform the absolute depth into relative depth for the ground truth depth maps in your training dataset? 
I ask this question because I want to transform the predicted relative depth map back to the absolute depth.
Thank you for your time!
